src/main/java/frc/robot/commands/Climber2022SetLeftSpeed.java

```diff
@@ -0,0 +1,43 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.logging.RobotLogManager;
+import frc.robot.subsystems.Climber2022;
+import java.util.function.Supplier;
+import org.apache.logging.log4j.Logger;
+
+public class Climber2022SetLeftSpeed extends CommandBase {
+  private static final Logger LOGGER = RobotLogManager.getMainLogger(Climber2022StopCMD.class.getName());
+
+  private final Climber2022 climber;
+  private final Supplier<Double> speed;
+
+  public Climber2022SetLeftSpeed(Climber2022 climber, Supplier<Double> speed) {
+    this.climber = climber;
+    this.speed = speed;
+
+    addRequirements(climber);
+  }
+
+  @Override
+  public void initialize() {
+  }
+
+
+  @Override
+  public void execute() {
+    climber.setLeftSpeed(speed.get());
+  }
+
+
+  @Override
+  public void end(boolean interrupted) {
+    climber.setLeftSpeed(0);
+  }
+
+
+  @Override
+  public boolean isFinished() {
+    return false;
+  }
+}
```

Comment:
Again, I'm not sure why this is `false` instead of `true`. I suspect you've used "when pressed" to start this and "when released" to stop it. I wonder if it would be better to use "whilePressed". (I think I've got the names right..., if not, ask me).